releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,10 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v3.3.0 - July, 13 2022
+* Added KVSet file content profile support.
+* Fixed a bug that caused an unexpected error when pushing properties from .properties file.

```

Comment:
It will be great to share a bit more details of the error so customers can tell whether it's relevant to their scenarios.